autoPyTorch/data/tabular_feature_validator.py

```diff
@@ -142,21 +152,24 @@ def comparator(cmp1: str, cmp2: str) -> int:
 
             if len(categorical_columns) > 0:
                 self.categories = [
-                    # We fit an ordinal encoder, where all categorical
+                    # We fit a one-hot encoder, where all categorical
```

Comment:
Uum, what do you mean by one-hot encoder here?
The corresponding index for each categorical in onehot encoding?
As far as I can see from this line, `list(range(len(cat)))` is not a one-hot encoder...